### DISCUSSION TITLE
How can I change the font size on the sidebar for section and contents in quarto?

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#10403] [Q&A]
### Description

````qmd
project:
  type: website
  resources: 
    - "*.html"

website:
  title: "Módulo II"
  site-url: https://macovibmodulo02.netlify.app
  navbar:
    right:
      - text: Inicio
        href: index.qmd
      - text: Sobre mi
        href: sobre_mi/sobre_mi.qmd
  
  sidebar:
    style: "floating"
    search: true
    background: light
    contents:
      - section: "Modelos lineales generales"
        contents:
          - text: "Regresión lineal simple"
            href: material_html/regresion_simple.qmd
          - text: "Regresión lineal múltiple"
            href: material_html/regresion_multiple.qmd
````

**Comment by mcanouil** [comment]
Use your favourite browser developer / inspect mode, look at the right CSS target and add a rule for that.

**Comment by ManuelSpinola** [comment]
Thank you! I will do that!

*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Sitio web institucional: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Sitio web personal: Sitio personal <https://mspinola-sitioweb.netlify.app>
Blog sobre Ciencia de Datos: Blog de Ciencia de Datos
<https://mspinola-ciencia-de-datos.netlify.app>
